frontend/server/s3.js

```diff
@@ -170,3 +171,15 @@ export async function getFile(key, destinationPath, anvilConfiguration) {
     throw new Error(message);
   }
 }
+
+export async function syncS3ToLocalDirectory(
+  s3Prefix,
+  localPath,
+  anvilConfiguration,
+) {
+  const client = getClient(anvilConfiguration);
+  const { sync } = new S3SyncClient({ client: client });
+
+  const s3Path = `s3://${anvilConfiguration.s3.bucket}/${s3Prefix}`;
+  await sync(s3Path, localPath, { del: true });
```

Comment:
If I load a pipeline that no longer exists in my s3 bucket (e.g. happens for me when I stop and restart orbstack, s3 is cleared), will the `del` param here clear my local history?

Reply:
Not tested, but I'd assume it would yeah

Reply:
Oh, I'm saying we don't want to do that. I would suggest we don't `del` without a more robust strategy for that.

Reply:
Ah ok, that's easy to fix